cg/server/api.py

```diff
@@ -467,6 +468,18 @@ def get_application_pipeline_limitations(tag: str):
     return jsonify([limitation.to_dict() for limitation in application_limitations])
 
 
+@BLUEPRINT.route("/orders")
+def get_orders():
+    """Return the latest orders."""
+    if request.args.get("limit"):
```

Comment:
I think we should wait with implementing pagination and filtering

Reply:
So you think it is preferable to remove it?

Reply:
Actually, maybe it is nice for dev purposes! You know best

Reply:
I'll keep it for now.